### DISCUSSION TITLE
💻A 'this code could be better' warning?

### DISCUSSION THREAD

**Discussion by MarleenGilsing** [hedyorg/hedy#5138] [Large Projects & Plans]
I was working on a project in level 15 and without thinking I used 'repeat 10 times' in my code. 
This is of course no problem, but when working on level 15 you should know how to use 'for i in range 1 to 10' instead.

Would it be possible to give the students a warning when they are using outdated codes and offer them a suggestion of how to change their codes?

**Comment by Felienne** [comment]
Lovely idea! We will discuss it in the meeting next week (or maybe in the hackathon?)
